test/unit/plugins/resourcelink.test.js

```diff
@@ -0,0 +1,36 @@
+const expect = require('expect');
+const parser = require('../../../src/cdoFlavoredParser');
+
+describe('resourcelink', () => {
+  describe('render', () => {
+    it('cannot render resourcelinks to html', () => {
+      const input = "[r some-slug]";
+      const output = parser.sourceToHtml(input);
+      expect(output).toEqual("<p>[r some-slug]</p>\n");
+    });
```

Comment:
why does this test case prove that you cannot render to html -- what would this output look like if you could?